### ISSUE TITLE
Draw Shapes 

### ISSUE THREAD

**Issue by denisices** [charlielee/boats-animator#271]
I would like to suggest shape drawing and free hand dawing. It would be easier to do an arc movement and plan ease in/out.

**Comment by charlielee** [comment]
Thanks Denis for creating this issue. I assume this drawings would overlay the live preview in "capture mode"?

**Comment by denisices** [comment]
Yes! To use as reference only and not appear in the final image.
Thank you!

**Comment by charlielee** [comment]
Tldraw would be an obvious choice for this. It has quite a strict non-commercial licence but that should be compatible with BA https://github.com/tldraw/tldraw https://tldraw.dev/community/license

**Comment by charlielee** [comment]
Had another suggestion for this before
